include/dpp/queues.h

```diff
@@ -70,32 +108,65 @@ enum http_error {
  * rate limit figures, and returned request body.
  */
 struct DPP_EXPORT http_request_completion_t {
-	/** @brief HTTP headers of response */
+	/**
+	 * @brief HTTP headers of response.
+	 */
 	std::multimap<std::string, std::string> headers;
-	/** @brief HTTP status, e.g. 200 = OK, 404 = Not found, 429 = Rate limited */
+
+	/**
+	 * @brief HTTP status (e.g. 200 = OK, 404 = Not found, 429 = Rate limited, etc).
+	 */
 	uint16_t status = 0;
-	/** @brief Error status (e.g. if the request could not connect at all) */
+
+	/**
+	 * @brief Error status (e.g. if the request could not connect at all).
```

Comment:
same anywhere where you put (e.g.) on the same line in the brief, there arent many